Frontiers_317_delete_products.py

```diff
@@ -0,0 +1,47 @@
+"""
+Working on a script to clean up demo store. delete all products without images.
+"""
+
+from woocommerce import API
+
+wcapi = API(
+    url="http://localhost:8888/mysite2",
+    consumer_key="<add consumer key>",
```

Comment:
get the credentials from env variable.
that way you don't have to modify the script to run it.
And it makes it easy to run in jenkins.
If you want go ahead and set it up in jenkins.
You can create same file in GitLab and have a job run the script too. In git lab you go to Settings > CI/CD > Variables to store your credentail variables.